internal/assert/string.go

```diff
@@ -0,0 +1,19 @@
+package assert
+
+import (
+	"strings"
+	"testing"
+)
+
+// StringContains asserts that the small string contains the smaller. Returns
```

Comment:
which one is the smaller and which one the small one?

```suggestion
// StringContains asserts that s contains substr. Returns
```

Reply:
I renamed the variables to match the names used on `strings.Contains and missed the godocs. Documentation is updated.